Framework/Algorithms/src/CompareWorkspaces.cpp

```diff
@@ -136,7 +137,7 @@ void CompareWorkspaces::init() {
                   "The name of the second input workspace.");
 
   declareProperty(
-      "Tolerance", 0.0,
+      "Tolerance", 1e-10,
```

Comment:
Can you revert this change as well since it's not needed for this PR.

Reply:
Got it, I will do it now.